pyproject.toml

```diff
@@ -115,21 +123,21 @@ select = [
     "BLE",  # flake8-blind-except
     "C4",  # flake8-comprehensions
     "D",  # pydocstyle
-#    "E",  # pycodestyle error
-#    "EM",  # flake8-errmsg
-#    "ERA",  # eradicate
-#    "F",  # Pyflakes
-#    "FA",  # flake8-future-annotations
+    "E",  # pycodestyle error
+    "EM",  # flake8-errmsg
+    "ERA",  # eradicate
+    "F",  # Pyflakes
+    "FA",  # flake8-future-annotations
 #    "FBT",  # flake8-boolean-trap
```

Comment:
purposefully skipping FBT for this PR.  Will be addressed in it's own at the end.